VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -135,17 +134,13 @@ public void onActivityCreated(Bundle savedInstanceState) {
             if ( courseData == null )
                 return;
 
-            String headerImageUrl = courseData.getCourse().getCourse_image(environment.getConfig());
-            NetworkImageView headerImageView = (NetworkImageView)getView().findViewById(R.id.header_image_view);
-            DisplayMetrics displayMetrics = getResources().getDisplayMetrics();
-            float screenWidth = displayMetrics.widthPixels;
-            float ideaHeight = screenWidth * 9 / 16;
-
-            headerImageView.setLayoutParams(new LinearLayout.LayoutParams(
-                LinearLayout.LayoutParams.MATCH_PARENT, (int) ideaHeight));
-            headerImageView.requestLayout();
-
-            headerImageView.setImageUrl(headerImageUrl, environment.getImageCacheManager().getImageLoader() );
+            final String headerImageUrl = courseData.getCourse().getCourse_image(environment.getConfig());
+            final ImageView headerImageView = (ImageView)getView().findViewById(R.id.header_image_view);
```

Comment:
This header view should be initialized along with the other views in `onCreateView()`. In fact they should all be injected via RoboGuice instead.